### ISSUE TITLE
Javascript in the Webview is not executed on iOS with new RN architecture from 13.0.0

### ISSUE THREAD

**Issue by louiszawadzki** [react-native-webview/react-native-webview#3003]
**Bug description:**
JS is not executed inside Webviews on iOS from `react-native-webview` 13.0.0.
Anything in `<script>` tags or `onclick` callbacks are not executed.

Version `12.4.0` or using the old RN architecture works fine.

**To Reproduce:**
- Create new RN app
- Run `yarn add react-native-webview` 
- Run `(cd ios && RCT_NEW_ARCH_ENABLED=1 pod install)`
- Paste this component into your App.tsx:

```ts
import WebView from 'react-native-webview';
import React from 'react';
import {
  View,
} from 'react-native';

function App(): JSX.Element {
  return (
      <View
        style={{
          marginTop: 20,
          height: 560,
          width: 300,
        }}>
        <WebView
          source={{
            html: `
            <html lang="en">
  <head>
    <script>alert('Loading WebView')</script>
  </head>

  <body>
        <button type="button" onclick="alert('clicked button')">Trigger alert</button>
  </body>
</html>`,
          }}
        />
      </View>
  );
}

export default App;

```

**Expected behavior:**
- An alert showing "Loading Webview" shows up
- When clicking on the button, "clicked button" shows up

**Screenshots/Videos:**

No alert show up.

**Environment:**
 - OS: iOS
 - OS version: tested in 16.4, 16.3
 - react-native version: 0.71.8
 - react-native-webview version: 13.0.0

**Comment by SvenNussgruber** [comment]
@louiszawadzki I ran into a similar issue on an older version (v11). Javascript events were just not being triggered anymore. It still worked on other dev's systems, so what fixed it in the end for me was recloning the repo 🤷‍♂️ hope this helps

**Comment by agent-0028** [comment]
I can confirm, this is a regression in 13.x.

We have been working towards integrating the WebView into our app for a while, I tried to upgrade to the latest and seems JS is not being evaluated at all.

Going back to 12.4.0 is working properly in this regard.

**Comment by billnbell** [comment]
Same for me - v13+ does not have Javascript while 12.4 does.

Who can look into this?

**Comment by TheAlmightyBob** [comment]
Looked into and published #3041. Thank you all for the report! The change is simple enough that I encourage you to try patching the library in the meantime.

**Comment by TheAlmightyBob** [comment]
Ah, I had mentioned this issue in the PR, but I guess I had too many other words in between "fixes" and the issue number, so the auto-linking/closing didn't work 😛
